java/src/com/swiftnav/sbp/client/MessageTable.java

```diff
@@ -173,6 +173,10 @@
 import com.swiftnav.sbp.settings.MsgSettingsReadByIndexDone;
 import com.swiftnav.sbp.settings.MsgSettingsRegister;
 import com.swiftnav.sbp.settings.MsgSettingsRegisterResp;
+import com.swiftnav.sbp.solution_meta.Gnssinputtype;
```

Comment:
We need to fix the java generator so that it correctly camel cases these embedded types.

Reply:
Good catch @silverjam ! Thanks!

Reply:
Done! 👍